Multi-Signal-Analyzer/led_control.h

```diff
@@ -1,35 +1,39 @@
+/** -------------------------------------------------------------
+ * @file led_control.h
+ * @brief LED control module interface
+ *
+ * @author    Vinicius Rafael Marques de Carvalho vinicius.carvalho@edge.ufal.br
+ * @version   v1.0
+ * @date      28/11/2025
+ * @copyright
+ *  ------------------------------------------------------------*/
+
 #ifndef LED_CONTROL_H
 #define LED_CONTROL_H
 
 #include "pico/stdlib.h"
 
-//Defines Gpios for the LED pins
+// Setting to the same pin for raspberry pi pico onboard led
+// but will change for the PCB board
+#define LED_CONNECTED_PIN    25
+#define LED_ENABLE_BLINK_PIN 2
 
-//Setting to the same pin for raspberry pi pico onboard led
-//but will change for the PCB board
-#define LED_CONNECTED_PIN 25
-#define LED_ENABLE_BLINK_PIN 2 
-
-typedef enum
-{
-    LED_STATUS_ERROR,     
-    LED_STATUS_CONNECTED, 
-    LED_STATUS_CAPTURING  
+typedef enum {
+	LED_STATUS_ERROR,
+	LED_STATUS_CONNECTED,
+	LED_STATUS_CAPTURING
 } led_status_t;
```

Comment:
Também documentamos o enum. Tanto acima dele (com o que ele representa por completo), como cada opção.

Reply:
Além disso, evitamos usar typedef, pq o que o tipo é, fica escondido. Para isso podemos fazer assim:
```suggestion
enum led_status {
	LED_STATUS_ERROR,
	LED_STATUS_CONNECTED,
	LED_STATUS_CAPTURING
};
```
Desta foram, na hora de usar faremos `enum led_status status;`